main/test/src/mill/define/CacherTests.scala

```diff
@@ -17,7 +17,7 @@ object CacherTests extends TestSuite{
   }
   object Middle extends Middle
   trait Middle extends Base{
-    def value = T{ super.value() + 2}
+    override def value = T{ super.value() + 2}
```

Comment:
Intellij IDEA is complaining about this line (preventing me from running the tests) but Mill can compile the code just fine :neutral_face:

Reply:
Yep, that's fine in mill, because there is a compiler plugin that makes `override` word optional

Reply:
Thanks for the explanation.
Do you know if there is a way to configure intellij IDEA in order to use this compiler plugin for this file ?

Reply:
Technically it should be possible, If I understand correctly, you need to include http://search.maven.org/#artifactdetails%7Ccom.lihaoyi%7Cmill-moduledefs_2.12%7C0.1.7%7Cjar in classpath and add "-P:auto-override-plugin". to scalac options.